content/docs-community/concepts/contributing-to-docs/terms/docs-repository/docs-repository.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Docs Repository'
+Description: 'The docs repository is where contributors can view, edit, and propose new content to be published to the Codecademy Docs site.'
+Subjects:
+  - 'Code Foundations'
+Tags:
+  - 'Documentation'
+  - 'Git'
+  - 'GitHub'
+CatalogContent:
+  - 'introduction-to-open-source'
+  - 'paths/code-foundations'
+---
+
+The content-base where Codecademy Docs is housed is the **`docs` repository** (or "repo") on [GitHub](https://github.com/Codecademy/docs). Contributors can view, edit, and propose new content for the repo. After the changes are merged into the repo's `main` branch, they will go live on the [Docs site](https://www.codecademy.com/resources/docs).
+
+## Forking the Repository
+
+When getting started, contributors should create a new fork of the `docs` repo. This should be done prior to changing any existing files or creating new ones in order to avoid merge conflicts.
+
+![A red stencil outlining the "Fork" link for the Codecademy Docs repo](https://raw.githubusercontent.com/Codecademy/docs/main/media/indicate_fork_link.png)
+
+The forked repo is sometimes regarded as the "remote repo" on GitHub. To make a changes on the local computer, a "local" version of the remote `docs` repo will need to be made with the following command:
+
+```pseudo
+git clone https://github.com/github_username/docs.git
+```
+
+The newly created `docs` folder (the local repo) can now be changed into. It contains all of the files and folders from the remote repo.
+
+## Issues
+
+First-time contributors are highly encouraged to visit the [Issues](https://github.com/Codecademy/docs/issues) tab of the `docs` repo. There, they can ask to be assigned to an open issue and begin writing a markdown file for a new [entry](https://www.codecademy.com/resources/docs/docs-community/entries) or make edits to an existing one.
+
+## Pull Requests
+
+Pull requests (PR) are used to propose new changes to be merged in the base `main` branch of the `docs` repo. After contributors are finished editing files in the PR branch of their forked repo (separate from the `main` branch), they are ready to [add](https://www.codecademy.com/resources/docs/git/add), [commit](https://www.codecademy.com/resources/docs/git/commit), and [push](https://www.codecademy.com/resources/docs/git/push) their changes and open a new PR.
```

Comment:
Instead of "the PR branch of their forked repo." Maybe say "their local repo". Otherwise the change in terminology might be confusing.